client/src/routes/favorites/index.js

```diff
@@ -0,0 +1,56 @@
+import { h, Component } from 'preact';
```

Comment:
With the way you are using this, bringing it in to the routes/profile component it should live in the components directory. Reserve the routes directory for components that are complete views made up of a collection of components like this one.